src/Entity/EncryptedFieldValueInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\field_encrypt\Entity\EncryptedFieldValueInterface.
+ */
+
+namespace Drupal\field_encrypt\Entity;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+
+/**
+ * Provides an interface defining an EncryptedFieldValue entity.
+ *
+ * @ingroup field_encrypt
+ */
+interface EncryptedFieldValueInterface extends ContentEntityInterface {
+
+  /**
+   * Get the encrypted field value.
+   *
+   * @return string
+   *   The encrypted value.
+   */
+  public function getEncryptedValue();
+
+  /**
+   * Set the encrypted field value.
+   *
+   * @param string $value
+   *   The encrypted value.
+   */
+  public function setEncryptedValue($value);
```

Comment:
I wonder if we should have a removeEncryptedValue() function if someone turns off Field Encrypt settings